projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/linked-cases/enums/write-linked-cases-field.enum.ts

```diff
@@ -0,0 +1,31 @@
+export enum LinkedCasesPages {
+  BEFORE_YOU_START,
+  NO_LINKED_CASES,
+  LINK_CASE,
+  UNLINK_CASE,
+  CHECK_YOUR_ANSWERS
+}
+
+export enum LinkedCasesErrorMessages {
+  ProposedCaseWithIn = 'Case can not be linked to the same case',
+  CaseNumberError = 'Case numbers must have 16 digits',
+  ReasonSelectionError = 'Select a reason why these cases should be linked',
+  SomethingWrong = 'Something went wrong, please try again later',
+  CaseCheckAgainError = 'Check the case number and try again',
+  CaseSelectionError = 'You need to propose at least one case',
+  CaseProposedError = 'This case has already been proposed',
+  CasesLinkedError = 'These cases are already linked',
+  UnlinkCaseSelectionError = 'Select a case to unlink before continue',
```

Comment:
@johnbenjamin-hmcts - change `continue` to `continuing` (correct grammar)

Reply:
@Daniel-Lam I have updated the code now.